package.json

```diff
@@ -20,6 +20,7 @@
     "eslint": "8.33.0",
     "eslint-config-next": "13.1.6",
     "next": "13.1.6",
+    "rc-switch": "^4.0.0",
```

Comment:
Please run `npm install` once and commit the changes the happen to the `package-lock.json`. Currently they are out of sync in this branch.